### PR TITLE
Fixes for Issue #10 and Issue #13

### PR DESCRIPTION
**Pull request by KevinAdams05**
Please review for style and formatting. I'm still getting used to the c++ / Haiku style. Thanks!

### CODE REVIEW

**Comment by KevinAdams05** [comment]
Oops, I didn't intend for both of these to be on the same pull request.  I was going to split them into separate requests, but I forgot that if I commit again it adds to the open pull request.

File: src/Minefield.cpp [hunk 1]
```diff
@@ -281,12 +313,36 @@ Minefield::FireSonar(const IntPoint &pt)
...
+Minefield::IsBoxBeingRevealed(const IntPoint &pt)
+{
+	if (GetState(pt) == BOX_MARKED
+		|| GetState(pt) == BOX_QUESTION
+		|| GetState(pt) == BOX_REVEALED
+		|| GetState(pt) == BOX_HIT
+		|| GetState(pt) == BOX_MINE
+		|| GetState(pt) == BOX_NOTMINE
+		|| GetState(pt) == BOX_INVALID)
+	{
```
Comment (waddlesplash) [inline src/Minefield.cpp]: Brace to previous line.

File: src/ScoreWindow.cpp [hunk 2]
```diff
@@ -95,20 +92,42 @@ void
...
 {
 	BString scoreLabel;
-	scoreLabel << gBestTimes[DIFFICULTY_BEGINNER].name
-				<< ", " << (int)gBestTimes[DIFFICULTY_BEGINNER].time << " seconds";
+
+	if (gBestTimes[DIFFICULTY_BEGINNER].name == "Anonymous" &&
+		(int)gBestTimes[DIFFICULTY_BEGINNER].time == 999){
+		scoreLabel = B_UTF8_ELLIPSIS;
+    }
+    else{
```
Comment (waddlesplash) [inline src/ScoreWindow.cpp]: Tab not spaces for indentation. Also these 2 lines should be one (`} else {`)

Comment (waddlesplash) [reply src/ScoreWindow.cpp]: Same goes for instances of similar code below.

File: src/ScoreWindow.cpp [hunk 3]
```diff
@@ -118,6 +137,9 @@ ScoreWindow::UpdateLabels(void)
 	right = MAX(right, fExpScore->Frame().right);
 	right += 20;
 
+	if(right < 200)
```
Comment (waddlesplash) [inline src/ScoreWindow.cpp]: Space after `if`

**Review by waddlesplash — COMMENTED** [review]
Looks pretty good!

**Comment by KevinAdams05** [comment]
Updated, thanks!

**Pull request merged**